config/route.go

```diff
@@ -71,3 +93,13 @@ func (r *Route) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 func (r *Route) Reset() {
 	r.index = 0
 }
+
+// Set returns the internal index of the route to 0
```

Comment:
This doesn't seem to match with what the method does. It's preferable for it to be in the format of `// Set takes x y z, and does a b c.`